tuts/create-your-first-substrate-chain/v2.0.0-rc2/interact.md

```diff
@@ -61,41 +61,72 @@ $ ./target/release/node-template --dev
 If the number after `finalized:` is increasing, your blockchain is producing new blocks and reaching
 consensus about the state they describe!
 
-## Start the Front End
+## Start the Front-End
 
-To interact with the local node we will use the Polkadot-js Apps user interface, often known as
-"Apps" for short. Despite the name, Apps will work with any Substrate-based blockchain including
-ours, not just Polkadot.
+To interact with your local node, we will use
+[the Substrate Developer Hub Front-End Template](https://github.com/substrate-developer-hub/substrate-front-end-template),
+which is a collection of UI components that have been designed with common uses cases in mind.
```

Comment:
```suggestion
which is a collection of UI components that have been designed with common use cases in mind.
```
…but now my eyes see and unsee hyphenation errors all the time. Is it "use-cases" maybe? :)

Reply:
LOL I know right! I had to second-guess myself several times on that one :sweat_smile:

Reply:
Your-eyes will never be-the-same